libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```
@@ -97,6 +97,7 @@ ion-title {
 
 ion-content {
   --background: transparent;
+  color: var(--kirby-modal-color);
```

Consider adding fallback:
```suggestion
  color: var(--kirby-modal-color, #{get-color('black')});
```